src/main/java/eu/fasten/vulnerabilityproducer/utils/patches/DiffParser.java

```diff
@@ -84,14 +86,15 @@
                 String t = hunk.split("\\+")[1].split(",")[0];
 
                 int fromLineNumber = Integer.parseInt(s.strip());
-                int toLineNumber = Integer.parseInt(t.strip());
+                int toLineNumber   = Integer.parseInt(t.strip());
```

Comment:
Remove this extra white space here.

Reply:
I wanted to align the two `fromLineNumber` and `toLineNumber`. Looks better imo 😁